.github/workflows/ci.yml

```diff
@@ -23,6 +25,13 @@ jobs:
     - name: Build E2E Runner
       shell: cmd
       run: dotnet.exe build src\Ztm.EndToEndTests
+    - name: Install main database
```

Comment:
I think this name quite hard to understand due to there is possibility that PostgreSQL will host additional database. So change this to `Install PostgreSQL` looks better.